src/open_inwoner/react/components/ActionList/ActionList.scss

```diff
@@ -0,0 +1,33 @@
+action-list {
+  margin-block-start: 24px;
+}
+
+.denhaag-action {
+  border-left: none !important;
+  border-right: none !important;
+  font-size: 14px;
+  font-family: var(--oip-typography-sans-serif-font-family);
+
+  // Use browser default focus-state until there is a design for it.
+  &:focus {
+    outline: revert !important;
```

Comment:
nice,
i still would like to see a hover-color style as in https://nl-design-system.github.io/denhaag/?path=/story/react-action--list too though.
We are using that very light-grey also in our  KVK search-select hover/active colors (`var(--oip-color-gray-lightest)`)